packages/next-css/commons-chunk-config.js

```diff
@@ -3,16 +3,18 @@ module.exports = (config, test = /\.css$/) => {
   config.plugins = config.plugins.map(plugin => {
     if (
       plugin.constructor.name === 'CommonsChunkPlugin' &&
-      (plugin.filenameTemplate === 'commons.js' || plugin.filenameTemplate === 'main.js')
+      typeof plugin.minChunks !== 'undefined'
```

Comment:
Lets keep the existing check, since otherwise this breaks 5.0/5.1

Reply:
Heh, but the existing check is what breaks 6

Reply:
We need to do both checks 🕵️

Reply:
I just tested it in Next 5.1 and it didn't seem to break it. Why exactly do you check for `commons.js` or `main.js`?